Sources/SwiftCurrent_SwiftUI/Views/ModifiedWorkflowView.swift

```diff
@@ -0,0 +1,165 @@
+//
+//  ModifiedWorkflowView.swift
+//  SwiftCurrent
+//
+//  Created by Tyler Thompson on 7/20/21.
+//  Copyright © 2021 WWT and Tyler Thompson. All rights reserved.
+//
+//  swiftlint:disable file_types_order
+
+import SwiftUI
+import SwiftCurrent
+
+@available(iOS 14.0, macOS 11, tvOS 14.0, watchOS 7.0, *)
+public struct ModifiedWorkflowView<Args, Wrapped: View, Content: View>: View {
+    @Binding private var isLaunched: Bool
+
+    let inspection = Inspection<Self>()
+    private let wrapped: Wrapped?
+    private let workflow: AnyWorkflow
+    private let launchArgs: AnyWorkflow.PassedArgs
+    private var onFinish = [(AnyWorkflow.PassedArgs) -> Void]()
+    private var onAbandon = [() -> Void]()
+
+    @ObservedObject private var model: WorkflowViewModel
+    @StateObject private var launcher: Launcher
+
+    public var body: some View {
+        if isLaunched {
+            if let body = model.body as? Content {
+                body
+                    .onReceive(model.onAbandonPublisher) { onAbandon.forEach { $0() } }
+                    .onChange(of: isLaunched) { if $0 { launch() } }
+                    .onReceive(inspection.notice) { inspection.visit(self, $0) }
+            } else {
+                wrapped.onReceive(inspection.notice) { inspection.visit(self, $0) }
```

Comment:
I'm not sure this works but if it does I'd like:
```suggestion
                wrapped?.onReceive(inspection.notice) { inspection.visit(self, $0) }
```
To make it clear that that is optional.